.github/workflows/release-nethermind.yml

```diff
@@ -85,11 +86,23 @@ jobs:
           ${{ github.workspace }}/${{ env.PACKAGE_DIR }}/
           ${{ env.SCRIPTS_PATH }}/
         key: packages-${{ steps.build-runner.outputs.COMMIT_HASH }}
+        
+  approval:
```

Comment:
How does this approval work?

Reply:
This is `trigger-publish` renamed to `approval`.

Reply:
Everything but Homebrew. Now it does as well.

Reply:
OK, it's good then. Can mark it as resolved.